library.properties

```diff
@@ -0,0 +1,9 @@
+name=Amperka RGBEncoder
+version=0.1.0
```

Comment:
```suggestion
version=1.0.0
```

Let’s start with a confident 1.0.0 guaranteeing backward compatibility within 1.x.x for customers

Reply:
OK